aws-ts-assume-role/README.md

```diff
@@ -0,0 +1,66 @@
+# AWS AssumeRole Example
+
+This example demonstrates how to use the AssumeRole functionality of the AWS provider in order to create resources in
+the security context of an IAM Role assumed by the IAM User running the Pulumi program.
+
+## Deploying the Example 
+
+These instructions assume you are familiar with running Pulumi programs written in TypeScript. Some other examples 
+
+### Part 1: Privileged Components
+
+The Pulumi program in `create-role` requires credentials with permissions to create an IAM User, an IAM Role, and assign
+an AWS Access Key to the user. The program creates a new, unprivileged user with no policies attached, and a role which
+specifies a trust policy allowing assumption by the unprivileged user. The role allows the `s3:*` actions on all 
+resources.
+
+You'll need to set the `create-role:unprivilegedUsername` configuration variable to the name of the unprivilged user, as
+well as the AWS region in which to operate.
+
+```bash
+$ cd create-role
+$ npm install
+$ pulumi stack init assume-role-create
+$ pulumi config set create-role:unprivilegedUsername james+unpriv@mydomain.com
+$ pulumi config set aws:region us-east-1
+$ pulumi up
+```
+
+The program can then be run with `pulumi up`. The outputs of the program tell you the ARN of the Role, and the Access 
+Key ID and Secret associated with the User:
+
+```
+$ pulumi stack output --json
+{
+    "accessKeyId": "AKIAI7JE74TLY2LOEIJA",
+    "secretAccessKey": "<redacted>",
+    "roleArn": "arn:aws:iam::<redacted>:role/allow-s3-management-ad477e6"
+}
+```
+
+### Part 2: Assuming the Role
+
+The Pulumi program in `assume-role` creates an S3 bucket after assuming the Role created in Part 1. It should be run
+with the unprivileged user credentials created in Part 1. This can be configured as follows, from the `assume-role`
+directory.
+
+```bash
+$ cd assume-role
+$ npm install
+$ export AWS_ACCESS_KEY_ID="$(cd ../create-role && pulumi stack output accessKeyId)"
```

Comment:
This could be `export AWS_ACCESS_KEY_UD="$(pulumi stack output -s assume-role-create accessKeyId)"`. Likewise below.

Reply:
I've fixed all three instance of this in the latest version of this.